packages/script.js

```diff
@@ -11,19 +11,18 @@
   const siteId = staticshieldsScript.getAttribute('data-site-id');
   const token = window.localStorage.getItem('staticshield-token');
 
-  if (token == null || undefined || !token) {
+  if (!token) {
     window.location.replace(
       `${BASE_URL}/p/${siteId}/?&redirecturl=${window.location.href
         .split('?')[0]
         .toString()}`
     );
+    return;
   }
 
   if (token) {
     const res = await fetch(
-      `https://staticshield.glitch.me/verify-token/?token=` +
-        decodeURIComponent(token)
-      // 'http://localhost:3000/api/verify-token/?token=' + token
+      `${BASE_URL}/api/verify-token?token=` + decodeURIComponent(token)
```

Comment:
I guess the `express` server on Glitch was added to workaround the CORS errors? The trailing slash here was the culprit of the errors.